phpstan-without-extension-baseline.neon

```diff
@@ -31,65 +31,85 @@ parameters:
 			path: test/StaticAnalysis/Test/ObjectProphecy/ProphesizeTest.php
 
 		-
-			message: "#^Call to an undefined method Prophecy\\\\Prophecy\\\\ObjectProphecy\\:\\:doubleTheNumber\\(\\)\\.$#"
-			count: 1
+			message: "#^Call to an undefined method Prophecy\\\\Prophecy\\\\ObjectProphecy\\<JanGregor\\\\Prophecy\\\\Test\\\\StaticAnalysis\\\\Src\\\\BaseModel\\>\\:\\:doubleTheNumber\\(\\)\\.$#"
+			count: 2
```

Comment:
I'm little bit curious why this count did increase. Not sure if I did something wrong in the baseline update, hope you can have a look at it if I did all correctly.

Reply:
that's not only a count increase. It shows an modified message too.

Reply:
The object was changed by me to have 0 errors when using. The extension. Not sure but I think when the run shows 0 errors when extension is enabled and show this errors when extension not enabled we should be fine.

Reply:
to me, that's fine indeed, based on the change you made.